lib/galaxy/tools/deps/views.py

```diff
@@ -125,5 +125,7 @@ def installable_resolvers(self):
         """
         return [index for index, resolver in enumerate(self._dependency_resolvers) if hasattr(resolver, "install_dependency") and not resolver.disabled ]
 
-    def get_requirements_status(self, requested_requirements):
+    def get_requirements_status(self, requested_requirements, installed_tool_dependencies=None):
+        for req in requested_requirements:
+            req['installed_tool_dependencies'] = installed_tool_dependencies
```

Comment:
Isn't this modification of the elements of `requested_requirements`  an unexpected side-effect?